gallery/src/components/NxDrawer/NxDrawerWithSubtitleOrDescriptionExample.tsx

```diff
@@ -0,0 +1,196 @@
+    <NxDrawer.Content>
+      <NxP>
+        Powder tiramisu gingerbread I love gummi bears I love. Lollipop gingerbread bonbon chupa chups cookie
+        I love dessert cake. Pie candy canes liquorice jelly beans sweet roll. Jelly candy donut cotton candy
+        halvah. Fruitcake halvah I love cheesecake I love I love. Wafer sweet sweet roll apple
+        pie jelly-o cheesecake candy I love.
+      </NxP>
+      <NxP>
+        Brownie dessert candy wafer macaroon. Marzipan dragée liquorice biscuit icing I love.
+        Wafer pastry sweet candy canes pie pie icing <strong>brownie</strong>. Wafer jelly cake bear claw I
+        love caramels. Pie jelly-o candy jelly beans icing. Sweet gingerbread pastry jelly bonbon danish icing.
+      </NxP>
+      <NxP>
+        Toffee tootsie roll danish chupa chups dessert cookie I love gummi bears wafer. Dragée
+        chocolate cake cake brownie carrot cake caramels chocolate cake. Halvah jelly-o tart wafer
+        jelly. Pastry <em>carrot</em> cake cake bear claw bear claw. Toffee jelly beans biscuit jelly cotton
+        candy marzipan liquorice. Brownie candy pie sweet roll powder danish dragée. Cake cookie
+        topping pastry carrot cake donut. Bear claw danish powder muffin bonbon cookie. I love I love
+        tootsie roll croissant chupa chups oat cake. Icing gingerbread pastry marshmallow pudding
+        brownie gingerbread marzipan.
+      </NxP>
+      <NxP>
+        Dragée pastry soufflé shortbread donut fruitcake. Ice cream tart bear claw I love
+        cotton candy marzipan cotton candy cake danish. Pie gingerbread marshmallow bear claw
+        halvah tiramisu cotton candy icing topping. Liquorice chupa chups dessert carrot cake
+        macaroon wafer. Marshmallow apple pie danish muffin cupcake icing dessert I love lemon
+        drops. Cupcake I <em>love</em> candy canes dragée croissant cookie chocolate muffin. Marshmallow
+        powder lollipop cotton candy bonbon lollipop liquorice chupa chups jelly-o. Biscuit I
+        love marzipan pastry pie ice cream chocolate bar dessert sweet. Cake topping cookie
+        chocolate pie cupcake. I love pastry donut croissant macaroon chocolate cake icing macaroon marshmallow.
+      </NxP>
+      <NxP>
+        Brownie candy pie sweet roll <strong>powder</strong> danish dragée. Cake cookie
+        topping pastry carrot cake donut. Bear claw danish powder muffin bonbon cookie. I love I love
+        tootsie roll croissant chupa chups oat cake. Icing gingerbread pastry marshmallow pudding
+        brownie gingerbread marzipan. I love candy canes dragée croissant cookie chocolate muffin. Marshmallow
+        powder lollipop cotton candy bonbon lollipop liquorice chupa chups jelly-o.
+      </NxP>
+    </NxDrawer.Content>
+  );
+
+  return (
+    <>
+      <header className="nx-global-header">
+        <NxBackButton href="#/pages/Drawer" targetPageTitle="Documentation" />
+        <div className="nx-global-header__actions">
```

Comment:
Why is this included in all of the examples? Seems extraneous

Reply:
> Why is this included in all of the examples? Seems extraneous

Removed here: 71d0a09ab86b8b7004da9f36b1cd7eeeb6614e4b